make.nix

```diff
@@ -36,11 +36,10 @@ let
 
   envScript = nixpkgs.stdenvNoCC.mkDerivation ({
     name = scrubbed.name + "-gen-bashrc";
-    script = ./gen.py;
```

Comment:
ah, it totally slipped my mind that this (and the other added drv attrs) could mess with the early stdenv phases 🤦 

hopefully `envInp` and `bashPrompt`/`bashPromptPrefix`/`bashPromptSuffix` don't collide with anything; I should probably just switch to interpolating them directly into `buildPhase` or into a copy of `gen.py` to be safe though

thanks for tracking this down!